clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -76,6 +76,67 @@ public void testMetadata() throws Exception {
         assertTrue("Update needed due to stale metadata.", metadata.timeToNextUpdate(time) == 0);
     }
 
+    private static void checkTimeToNextUpdate(long refreshBackoffMs, long metadataExpireMs) {
+        Metadata metadata = new Metadata(refreshBackoffMs, metadataExpireMs);
+        long now = 10000;
+
+        // Metadata timeToNextUpdate is implicitly relying on the premise that the currentTimeMillis is always
+        // larger than the metadataExpireMs or refreshBackoffMs.
+        // It won't be a problem practically since all usages of Metadata calls first update() immediately after
+        // it's construction.
+        assert now > metadataExpireMs;
+        assert now > refreshBackoffMs;
```

Comment:
Why not use JUnit assertions? We generally avoid Java level assertions as they require JVM arguments to enable.